pkgs/servers/komga/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, stdenvnoCC
+, fetchurl
+, makeWrapper
+, jdk11_headless
+}:
+
+stdenvnoCC.mkDerivation rec {
```

OfBorg caught this, but not in the way I'd expect (erroring).

```suggestion
stdenvNoCC.mkDerivation rec {
```